### PR TITLE
Fixes for Windows cross-compilation

### PR DESCRIPTION
**Pull request by bilke**
In preparation for building on a new Jenkins Docker slave (will be available after some WKDV-work).

### CODE REVIEW

File: FileIO/CMakeLists.txt [hunk 1]
```diff
@@ -61,6 +61,12 @@ target_link_libraries(FileIO INTERFACE
 )
 if(QT4_FOUND)
 	target_link_libraries(FileIO PUBLIC Qt4::QtXml Qt4::QtXmlPatterns)
+	if(WIN32 AND CMAKE_CROSSCOMPILING)
+		find_package(OpenSSL)
+		if(OPENSSL_FOUND)
```
Comment (chleh) [inline FileIO/CMakeLists.txt]: Just curious: Why do we need OpenSSL?

**Comment by chleh** [comment]
Looks good to me, but I don't know much about it. :+1:

File: FileIO/CMakeLists.txt (hunk 1 above)
Comment (bilke) [inline FileIO/CMakeLists.txt]: We don't need SSL but the prebuilt Qt package from http://pkg.mxe.cc is built against it which means that `QtNetwork` depends on it (and `QtXmlPatterns` (containing the XML validator) depends on `QtNetwork`).

**Comment by TomFischer** [comment]
Looks good to me, too.

**Pull request merged**